.github/workflows/main-pr.yml

```
@@ -190,3 +198,38 @@ jobs:
+    name: Build and Publish (Playground)
+    needs: test-playground
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - uses: actions/checkout@v6
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

actionlint (1.7.12)

[error] the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)